Dfe.Academies.Academisation.Domain.Core/ProjectAggregate/ProjectNote.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Dfe.Academies.Academisation.Domain.Core.ProjectAggregate
+{
+	public class ProjectNote
+	{
+		public ProjectNote(string? subject,
+						   string? note,
```

Comment:
The indentation has gone a bit funny here.

Reply:
![image](https://user-images.githubusercontent.com/89836563/207605073-aaabccc6-7575-4dfb-8ffa-7056e9174305.png)

The "joys" of using tabs for indentation.